### PR TITLE
namePrefix fix for `ProfileInfo.fullName`

### PR DESCRIPTION
**Pull request by zwebfek**

### CODE REVIEW

File: src/ProfileInfo.coffee [hunk 1]
```diff
@@ -79,7 +79,11 @@ class root.ProfileInfo
 		# @final
 		# @type String
 		###
-		@fullName = -> "#{@firstName()} #{@namePrefix()} #{@lastName()}"
+		@fullName = ->
+			if @namePrefix() == null
```
Comment (lieuwex) [inline src/ProfileInfo.coffee]: Can you replace `==` with `is` here?

**Comment by lieuwex** [comment]
Failing tests have nothing to do with this PR, it's good.

**Comment by lieuwex** [comment]
Thank you!

**Comment by zwebfek** [comment]
Great, thanks!

**Pull request merged**